Teensy/src/gpio/gpio.cpp

```diff
@@ -53,12 +57,42 @@ bool updateSensorInfo(sensorData_s *pSensorData)
     return false;
 }
 
+
 static WheelSpeed_s gpio_getWheelSpeed()
 {
     WheelSpeed_s wheelSpeed = {};
     return wheelSpeed;
 }
 
+    // setup for GPS readings
```

Comment:
All initialisation stuff should be in gpio_init()

This includes GPS.begin(), GPS.sendCommand(PMTK_SET_NMEA_OUTPUT_RMCGGA).....

SoftwareSerial and Adafruit_GPS objects should be defined at top of file, around line 33